samples/inputs/textarea/overview/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-input-overview",
+  "description": "This project provides example of Input Overview using Infragistics React components",
```

Comment:
In each `package.json` file change the `name` and `description` for each sample.

 ```
"name": "react-text-are-overview",
 "description": "This project provides example of Text Area Overview using Infragistics React components"
```